doc/lua_api.md

```diff
@@ -259,7 +259,18 @@ time, if necessary. (See [`Settings`])
 
 Media files (textures, sounds, whatever) that will be transferred to the
 client and will be available for use by the mod and translation files for
-the clients (see [Translations]).
+the clients (see [Translations]). Accepted name characters are:
```

Comment:
```suggestion
the clients (see [Translations]). Accepted characters for names are:
```

Let's try like this (since it doesn't see my other commit)